pkgs/top-level/all-packages.nix

```diff
@@ -511,13 +511,13 @@ with pkgs;
 
   avfs = callPackage ../tools/filesystems/avfs { };
 
-  awscli = pythonPackages.awscli; # Should be moved out of python-packages.nix
+  awscli = callPackage ../tools/admin/awscli { };
 
   awsebcli = callPackage ../tools/virtualization/awsebcli {};
 
   awslogs = callPackage ../tools/admin/awslogs { };
 
-  aws_shell = python2Packages.aws_shell; # Should be moved out of python-packages.nix
+  aws_shell = callPackage ../tools/admin/aws_shell { };
```

Comment:
It is better to use `pythonPackages.callPackage`, and list python dependencies in the arguments of the function.

Reply:
I agree. This is now changed.